kissmp-bridge/src/voice_chat.rs

```diff
@@ -29,75 +34,74 @@ fn search_config(
 }
 
 // Too much repetetive code, lol
+// It's _less_ repetitive now, sorta.
```

Comment:
This comment is useless and doesn't provide any information. It'll make no sense to a person reading this code